Packs/ServiceNow/Integrations/ServiceNowv2/ServiceNowv2.yml

```diff
@@ -72,6 +72,18 @@ configuration:
   name: insecure
   required: false
   type: 8
+- defaultvalue: 'false'
+  display: Close XSOAR Incident
```

Comment:
need to improve display names here and perhaps also add tooltip to elborate its related to the mirror flow, please work with @yaron-libman on that